src/react/components/pages/signIn/signInForm.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Form, { FormValidation, Widget } from "react-jsonschema-form";
+import { ISignIn } from "../../../../models/applicationState";
+
+const formSchema = require("./signInForm.json");
+const uiSchema = require("./signInForm.ui.json");
+
+
+// Interface for properties
+export interface ISignInFormProps extends React.Props<SignInForm> {
+    signin: ISignIn; // described in models/applicationState.ts
+    onSubmit: (signin: ISignIn) => void; // submission function, has signin form as parameter
+}
+
+// Interface for state
+export interface ISignInFormState {
+    formSchema: any;
+    uiSchema: any;
+    formData: ISignIn;
+}
+  
+    const customFormats = {
```

Comment:
Do you think it would be valuable to have it in some separate file? I don't know if we will ever have another email input (register form?) but it might be useful to have it in some other places, like helpers directory etc